content/3_Silver/Func_Graphs.mdx

```diff
@@ -390,66 +396,84 @@ import java.util.*;
 
 public class badge
```

Comment:
Why is the focus problem "badge" at the top of the page and not right above the solution?